ci_framework/roles/openshift_login/tasks/try_login.yml

```diff
@@ -34,8 +34,9 @@
   environment:
     PATH: "{{ cifmw_path }}"
     KUBECONFIG: "{{ cifmw_openshift_login_kubeconfig }}"
-  ansible.builtin.command:
-    cmd: >-
+  ci_script:
+    output_dir: "{{ cifmw_openshift_login_basedir }}/artifacts"
```

Comment:
is that location actually created in the role?

Reply:
Is it necessary? Because the `ci_script` module fails if this folder doesn't exist, isn't? Here is the artifact of this task: https://logserver.rdoproject.org/08/508/4ccac5a1eefc11ae08101c7a9054dd69f9f5da3f/github-check/cifmw-molecule-openshift_login/b236ea0/ci-framework-data/artifacts/ (Maybe I'm wrong.)

Reply:
Hi @adrianfusco 

I think Cedric means like [1] so while the ci_script module will fail if the directory doesn't exist it would be best to ensure it does exist. 

[1] https://github.com/lewisdenny/ci-framework/blob/9c8cdc4609a8252bd42983d440c2ea23e4554505/ci_framework/roles/libvirt_manager/tasks/main.yml#L17-L18

Reply:
Hi @lewisdenny ack. I'll update the PR

Reply:
Thanks for addressing this, lgtm 👍🏻